server/server.go

```diff
@@ -293,6 +304,19 @@ func (w *otherWorker) Do(job interface{}) interface{} {
 		w.s.stats.logRequestDuration(requestBegin)
 		return makeRespondResponse(pkt.ID, res)
 
+	case protocol.OpRPC:
+		codec := newServerCodec(pkt.Payload)
+
+		err := w.s.dispatcher.ServeRequest(codec)
+		if err != nil {
+			log.Errorf("Worker %v: ServeRPC: %v", w.name, err)
+			w.s.stats.logInvalid(requestBegin)
+			return makeErrResponse(pkt.ID, protocol.ErrInternal)
+		}
+
+		w.s.stats.logRequestDuration(requestBegin)
+		return makeRespondResponse(pkt.ID, codec.response)
+
```

Comment:
When you're configuring the server, where you'd call SetGetCert / SetSeal, you'd also call RegisterRPC(Obj{}). This would allow clients to call methods on Obj (and these methods can modify the fields of Obj) through the keyless protocol.